distributed/dashboard/scheduler.py

```diff
@@ -1701,3 +1680,33 @@ def listen(self, *args, **kwargs):
         ]
 
         self.server._tornado.add_handlers(r".*", handlers)
+
+
+applications = {
+    "/system": systemmonitor_doc,
+    "/stealing": stealing_doc,
+    "/workers": workers_doc,
+    "/events": events_doc,
+    "/counters": counters_doc,
+    "/tasks": tasks_doc,
+    "/status": status_doc,
+    "/profile": profile_doc,
+    "/profile-server": profile_server_doc,
+    "/graph": graph_doc,
+    "/individual-task-stream": individual_task_stream_doc,
+    "/individual-progress": individual_progress_doc,
+    "/individual-graph": individual_graph_doc,
+    "/individual-profile": individual_profile_doc,
+    "/individual-profile-server": individual_profile_server_doc,
+    "/individual-nbytes": individual_nbytes_doc,
+    "/individual-cpu": individual_cpu_doc,
+    "/individual-nprocessing": individual_nprocessing_doc,
+    "/individual-workers": individual_workers_doc,
+}
+
+try:
+    import pynvml  # noqa: 1708
+except ImportError:
+    pass
+else:
+    from . import nvml  # noqa: 1708
```

Comment:
I don't really understand, why are the imports needed? They don't seem to be used anywhere in this PR.

Reply:
They import the `nvml` module, which modifies the `applications` dictionary as a side effect.

Reply:
I see, thanks for clarifying!